modules/zivid/frame_2d.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, internal_frame_2d):
         Args:
             internal_frame_2d: internal 2D frame
 
+        Raises:
+            TypeError: unsupported type provided for internal 2d frame
+
```

Comment:
Is this a requirement from the updated linter?

Reply:
Yes, darglint requires this for the docs

Reply:
Nice :)